autogpt_platform/backend/backend/data/graph.py

```diff
@@ -123,6 +122,12 @@ def stripped_for_export(self) -> "NodeModel":
                 stripped_node.input_default, self.block.input_schema.jsonschema()
             )
 
+        if (
+            stripped_node.block.block_type == BlockType.INPUT
+            and "value" in stripped_node.input_default
+        ):
+            stripped_node.input_default["value"] = ""
+
```

Comment:
This is coming from marketplace download: https://github.com/Significant-Gravitas/AutoGPT/pull/9640/files#diff-b3cb5f1b999df920da4287781a6b51b1f65bf63622cfd010fcda9d4d1625d19bL202